Dockerfile

```diff
@@ -5,7 +5,7 @@ ENV APP_HOME /srv/circle_dropper/current
 
 RUN apk update && \
     apk upgrade && \
-    apk add build-base && \	
+    apk add build-base ruby-irb && \
```

Comment:
Idk Jake, we kinda want to keep our containers as small as possible.  Why would we add this remarkably useful tool that we need constantly when we could keep it 2MB smaller?